doc/nmstate.service.8.in

```diff
@@ -9,8 +9,20 @@ nmstate\&.service
 nmstate\&.service invokes \fBnmstatectl service\fR command which
 apply all network state files ending with \fB.yml\fR in
 \fB/etc/nmstate\fR folder.
-The applied network state file will be copied to new file with \fB.applied\fR
-postfix to prevent repeated applied on next service start.
+By default, the network state file will be renamed with the suffix
+\fB.applied\fR after applying it. This is to prevent it being applied again
+when the service runs next.
+
+With \fB/etc/nmstate/nmstate.conf\fR holding below content:
+
+\fB
+[service]
+delete_applied = false
```

Comment:
The name is weird since the `.applied` are the one that are going to be always there, let's call this `keep_state_file_after_apply=true`

Or we can have an enum, in case we have other mechanism in the future:
```
applied_file_generation=copy_state
applied_file_generation=move_state
applied_file_generation=checksum_state
```